wpeverest/everest-forms
# Feature - Brevo module compatibility

### All Submissions:






### Changes proposed in this Pull Request:


Previously, To use the Everest FormsSendinBlue(Now Brevo) we had to install the everest forms brevo addons. This PR adds the flexibility by including it in the module system under the features tab. Now, there is no need to install the plugin you can just hit the toggle button to enable the module or feature to use.

Closes # .

### How to test the changes in this Pull Request:

1. Go to Everest Forms Dashboard
2. Go to features tab and select the **Brevo** feature and click on the toggle button to enable the module
3. Now small **Settings** toggle button will appear.
4. Now enter the **API Key** and **Account nickname**
5. Click on the **Save** button.
6. Now go to **Forms Settings>Integration>Brevo**.
7. And create the connection by clicking on **Add connection** button
8. Select the account list and map the fields accordingly.
9. Finally, fill up the form and check whether it is working accordingly or not.

**Note**: Here please do check the backward compatibility and Double optin feature.
PR Dependency: [Brevo Dependency](https://github.com/wpeverest/everest-forms-pro/pull/900)

### Types of changes:




### Other information:




### Changelog entry

> Feature - Brevo module compatibility
